src/main/java/com/redhat/rhjmc/containerjfr/net/HttpServer.java

```diff
@@ -19,18 +24,92 @@
     private final Vertx vertx;
     private final HandlerDelegate<HttpServerRequest> requestHandlerDelegate = new HandlerDelegate<>();
     private final HandlerDelegate<ServerWebSocket> websocketHandlerDelegate = new HandlerDelegate<>();
+    private final boolean sslEnabled;
 
     private final io.vertx.core.http.HttpServer server;
-    
 
-    HttpServer(NetworkConfiguration netConf, Logger logger) {
+    private static final String KEYSTORE_PATH_ENV = "KEYSTORE_PATH";
+    private static final String KEYSTORE_PASS_ENV = "KEYSTORE_PASS";
+    private static final String KEY_PATH_ENV = "KEY_PATH";
+    private static final String CERT_PATH_ENV = "CERT_PATH";
+
+    HttpServer(NetworkConfiguration netConf, Environment env, Logger logger) {
         this.netConf = netConf;
         this.logger = logger;
         this.vertx = Vertx.vertx();
-        this.server = vertx.createHttpServer(new HttpServerOptions()
+
+        HttpServerOptions options = new HttpServerOptions()
                 .setCompressionSupported(true)
-                .setLogActivity(true)
-        );
+                .setLogActivity(true);
+        this.sslEnabled = setSslOptions(env, options);
```

Comment:
Is the return value here useful? It seems like `sslEnabled` should always have the same value as `options.isSsl()`.

Reply:
This is outdated. Such a field is no longer used.